scripts/ssl_legacysurvey/ssl_legacysurvey.py

```diff
@@ -92,7 +92,7 @@ class SSLLegacySurvey(datasets.GeneratorBasedBuilder):
             name="stein_et_al",
             version=VERSION,
             data_files=DataFilesPatternsDict.from_patterns(
-                {"train": ["*/healpix=*/*.hdf5"]}
+                {"train": ["north/healpix=*/*.hdf5"]}
```

Comment:
not `ssl_legacysurvey`?

Reply:
no, it's called north

Reply:
because in principle we could have also added a 'south'